InternetHospital.DataAccess/AppContextConfiguration/RoleConfiguration.cs

```diff
@@ -11,26 +13,35 @@ public class RoleConfiguration
     {
         public static async Task InitializeAsync(RoleManager<IdentityRole<int>> roleManager)
         {
-            const string PATIENT = "Patient";
-            const string DOCTOR = "Doctor";
-            const string MODERATOR = "Moderator";
-            const string ADMIN = "Admin";
-         
-            if (await roleManager.FindByNameAsync(PATIENT) == null)
+            try
             {
-                await roleManager.CreateAsync(new IdentityRole<int>(PATIENT));
-            }
-            if (await roleManager.FindByNameAsync(DOCTOR) == null)
-            {
-                await roleManager.CreateAsync(new IdentityRole<int>(DOCTOR));
+                const string PATH = @"..\InternetHospital.DataAccess\AppContextConfiguration\RoleConfigurationJSON.json";
+
+                string jsonString = File.ReadAllText(PATH);
```

Comment:
I don't think that it's good to separate declarations of variables by empty line if they are from the same block.

Reply:
Done